### PR TITLE
feat: Added screenshot options

### PR DESCRIPTION
**Pull request by bitsandfoxes**
Adding options to control screenshot quality back in.
The resolution is controlled via an enum.
- Full - capture "as is" in full resolution
- High - capture in half the resolution `Default`
- Medium - capture in one-third of the resolution
- Low - capture in a quarter of the resolution

JPG compression gets controlled via slider separately.

Screenshots

### CODE REVIEW

File: src/Sentry.Unity/SentryUnityOptions.cs [hunk 1]
```diff
@@ -70,6 +70,13 @@ public new CompressionLevelWithAuto RequestBodyCompressionLevel
         /// </summary>
         public bool AttachScreenshot { get; set; } = false;
 
+        /// <summary>
+        /// The quality of the attached screenshot
+        /// </summary>
+        public ScreenshotQuality ScreenshotQuality { get; set; } = ScreenshotQuality.High;
+
+        public int ScreenshotCompression { get; set; } = 75;
```
Comment (bruno-garcia) [inline src/Sentry.Unity/SentryUnityOptions.cs:78]: Missions docs

**Review by bruno-garcia — APPROVED** [review]
Tests?

**Comment by bitsandfoxes** [comment]
> Tests?

I'll follow up on that.

**Comment by oddgames-david** [comment]
Still not the right direction. MaxHeight is the best solution, means you can specify 480 and all screenshots are either 480 or below.

If I specify half resolution to stop 2k screenshots my 480's become unusable at half.

**Comment by bitsandfoxes** [comment]
> If I specify half resolution to stop 2k screenshots my 480's become unusable at half.

Fair point.

**Pull request merged**